third_party/mustache/mustache.js

```diff
@@ -5,16 +5,7 @@
 
 /*global define: false Mustache: true*/
 
-(function defineMustache (global, factory) {
-  if (typeof exports === 'object' && exports && typeof exports.nodeName !== 'string') {
-    factory(exports); // CommonJS
-  } else if (typeof define === 'function' && define.amd) {
-    define(['exports'], factory); // AMD
-  } else {
-    global.Mustache = {};
-    factory(Mustache); // script, wsh, asp
-  }
-}(this, function mustacheFactory (mustache) {
+function mustacheFactory (mustache) {
```

Comment:
@choumx @erwinmombay @jridgewell Is it reasonable to remove the UMD wrapper in this file to get single-pass compile to work?

Reply:
It's fine. As discussed, please update the README.md with the local modifications to this file.

Or even better, use the `*.patch` system that third_party/caja uses that @danielrozenberg wrote. :)

Reply:
Done, and done.